llvm/lib/ObjectYAML/XCOFFYAML.cpp

```diff
@@ -127,6 +127,16 @@ void ScalarEnumerationTraits<XCOFF::StorageMappingClass>::enumeration(
 #undef ECase
 }
 
+void ScalarEnumerationTraits<XCOFF::SymbolType>::enumeration(
```

Comment:
I forget how this enum stuff works exactly. Does it allow specification as a number directly too? For testing certain edge case behaviour in ELF code, it is often useful to be able to specify a value outside the range possible with the known enum values.

Reply:
I don't know how to specify a number rather than a string (denoting the enumeration type).  Generating invalid object files isn't always easy.

Reply:
The point of yaml2obj is to be able to generate these invalid objects. In many other cases , you can specify both `Field: SOME_ENUM` and `Field: 42`. I think you  usually have to add a line like this: https://github.com/llvm/llvm-project/blob/4f62a183d9101a90d62e2a278d0513e9d2afba43/llvm/lib/ObjectYAML/ELFYAML.cpp#L55